crime-commons-classes/build.gradle

```diff
@@ -1,12 +1,15 @@
-
 plugins {
+    id "com.javiersc.semver.project" version "0.5.0"
     id 'java'
```

Comment:
Can we clean this up a little?
This plugin can be removed as it's defined in the parent `build.gradle`.
The `java` closure can also be removed.